newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -728,15 +710,29 @@ export default class AuthenticatedUserProvider extends React.Component<
         ),
       });
     } catch (authError) {
-      this.setState({ authError });
+      this.setState({
+        authError,
+        authenticatedUser: {
+          ...this.state.authenticatedUser,
+          authenticationError: authError,
```

Comment:
it feels like we're duplicating the error & loading prop in 2 places. Can't we just have it in 1 place? This will lead to errors